### ISSUE TITLE
[production][delayed_job#StagingProjectAcceptJob] <status code="400">
  <summary>Package python-orangebox is controlled by obs-scm</summary>
</stat...

### ISSUE THREAD

**Issue by eduardoj** [openSUSE/open-build-service#12222]
[See this exception on Errbit](http://errbit.opensuse.org/apps/5620ca2bdc71fa00b1000000/problems/620d892284bf46054d088763)
# &lt;status code=&quot;400&quot;&gt;
  &lt;summary&gt;Package python-orangebox is controlled by obs-scm&lt;/summary&gt;
&lt;/status&gt;
 #
### Where ###
delayed_job#StagingProjectAcceptJob

### Occured ###
Feb 16 11:30pm

### Similar ###
1

## Params ##
~~~
{
  &quot;active_job&quot; =&gt; {
    &quot;arguments&quot; =&gt; [{&quot;project_id&quot;=&gt;972795, &quot;user_login&quot;=&gt;&quot;dimstar_suse&quot;, &quot;_aj_symbol_keys&quot;=&gt;[&quot;project_id&quot;, &quot;user_login&quot;]}],
    &quot;enqueued_at&quot; =&gt; &quot;2022-02-16T23:29:56Z&quot;,
    &quot;exception_executions&quot; =&gt; {},
    &quot;executions&quot; =&gt; 0,
    &quot;job_class&quot; =&gt; &quot;StagingProjectAcceptJob&quot;,
    &quot;job_id&quot; =&gt; &quot;eb46e9c9-b8e0-45b8-ad8c-d9544cf488ef&quot;,
    &quot;locale&quot; =&gt; &quot;en&quot;,
    &quot;priority&quot; =&gt; nil,
    &quot;provider_job_id&quot; =&gt; nil,
    &quot;queue_name&quot; =&gt; &quot;staging&quot;,
    &quot;timezone&quot; =&gt; &quot;UTC&quot;
  },
  &quot;attempts&quot; =&gt; 0,
  &quot;failed_at&quot; =&gt; nil,
  &quot;handler&quot; =&gt; &quot;--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: StagingProjectAcceptJob\n  job_id: eb46e9c9-b8e0-45b8-ad8c-d9544cf488ef\n  provider_job_id: \n  queue_name: staging\n  priority: \n  arguments:\n  - project_id: 972795\n    user_login: dimstar_suse\n    _aj_symbol_keys:\n    - project_id\n    - user_login\n  executions: 0\n  exception_executions: {}\n  locale: en\n  timezone: UTC\n  enqueued_at: &#39;2022-02-16T23:29:56Z&#39;\n&quot;,
  &quot;id&quot; =&gt; 678583449,
  &quot;last_error&quot; =&gt; nil,
  &quot;locked_at&quot; =&gt; &quot;2022-02-16T23:30:35.000Z&quot;,
  &quot;locked_by&quot; =&gt; &quot;delayed_job.1060 host:api pid:13461&quot;,
  &quot;priority&quot; =&gt; 0,
  &quot;queue&quot; =&gt; &quot;staging&quot;,
  &quot;run_at&quot; =&gt; &quot;2022-02-16T23:29:56.000Z&quot;
}
~~~

## Session ##
~~~
{}
~~~

## Backtrace ##
~~~
127:  app/lib/backend/connection.rb -> **handle_response**
113:  app/lib/backend/connection.rb -> **put_or_post**
42:  app/lib/backend/connection.rb -> **post**
56:  app/lib/backend/connection_helper.rb -> **http_post**
99:  app/lib/backend/api/sources/package.rb -> **branch**
15:  app/lib/backend/remember_location.rb -> **call**
15:  app/lib/backend/remember_location.rb -> **block in singleton_method_added**
98:  app/models/bs_request_action_submit.rb -> **execute_accept**
535:  app/models/bs_request.rb -> **block in changestate_accepted**
88:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/relation/delegation.rb -> **each**
88:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/relation/delegation.rb -> **each**
534:  app/models/bs_request.rb -> **changestate_accepted**
561:  app/models/bs_request.rb -> **block in change_state**
88:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/locking/pessimistic.rb -> **block in with_lock**
320:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/connection_adapters/abstract/database_statements.rb -> **block in transaction**
319:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/connection_adapters/abstract/transaction.rb -> **block in within_new_transaction**
26:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb -> **block (2 levels) in synchronize**
25:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb -> **handle_interrupt**
25:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb -> **block in synchronize**
21:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb -> **handle_interrupt**
21:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/concurrency/load_interlock_aware_monitor.rb -> **synchronize**
317:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/connection_adapters/abstract/transaction.rb -> **within_new_transaction**
320:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/connection_adapters/abstract/database_statements.rb -> **transaction**
209:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/transactions.rb -> **transaction**
290:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/transactions.rb -> **transaction**
86:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/locking/pessimistic.rb -> **with_lock**
558:  app/models/bs_request.rb -> **change_state**
18:  app/jobs/staging_project_accept_job.rb -> **block in accept**
88:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/relation/delegation.rb -> **each**
88:  /GEM_ROOT/gems/activerecord-6.1.4.6/lib/active_record/relation/delegation.rb -> **each**
14:  app/jobs/staging_project_accept_job.rb -> **accept**
8:  app/jobs/staging_project_accept_job.rb -> **block in perform**
864:  app/models/user.rb -> **run_as**
7:  app/jobs/staging_project_accept_job.rb -> **perform**
48:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/execution.rb -> **block in perform_now**
117:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
328:  /GEM_ROOT/gems/i18n-1.10.0/lib/i18n.rb -> **with_locale**
9:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/translation.rb -> **block (2 levels) in &lt;module:Translation&gt;**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **instance_exec**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
66:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/core_ext/time/zones.rb -> **use_zone**
9:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/timezones.rb -> **block (2 levels) in &lt;module:Timezones&gt;**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **instance_exec**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
21:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/instrumentation.rb -> **block in instrument**
203:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/notifications.rb -> **block in instrument**
24:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/notifications/instrumenter.rb -> **instrument**
203:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/notifications.rb -> **instrument**
31:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/instrumentation.rb -> **instrument**
14:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/instrumentation.rb -> **block (2 levels) in &lt;module:Instrumentation&gt;**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **instance_exec**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
22:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/logging.rb -> **block in tag_logger**
99:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/tagged_logging.rb -> **block in tagged**
37:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/tagged_logging.rb -> **tagged**
99:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/tagged_logging.rb -> **tagged**
22:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/logging.rb -> **tag_logger**
15:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/logging.rb -> **block (2 levels) in &lt;module:Logging&gt;**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **instance_exec**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
137:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **run_callbacks**
47:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/execution.rb -> **perform_now**
25:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/execution.rb -> **block in execute**
117:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
47:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/railtie.rb -> **block (4 levels) in &lt;class:Railtie&gt;**
91:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/execution_wrapper.rb -> **wrap**
72:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/reloader.rb -> **block in wrap**
91:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/execution_wrapper.rb -> **wrap**
71:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/reloader.rb -> **wrap**
46:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/railtie.rb -> **block (3 levels) in &lt;class:Railtie&gt;**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **instance_exec**
126:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **block in run_callbacks**
137:  /GEM_ROOT/gems/activesupport-6.1.4.6/lib/active_support/callbacks.rb -> **run_callbacks**
23:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/execution.rb -> **execute**
42:  /GEM_ROOT/gems/activejob-6.1.4.6/lib/active_job/queue_adapters/delayed_job_adapter.rb -> **perform**
81:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/backend/base.rb -> **block in invoke_job**
10:  /GEM_ROOT/gems/airbrake-9.4.3/lib/airbrake/delayed_job.rb -> **block (2 levels) in &lt;class:Airbrake&gt;**
79:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block (2 levels) in add**
61:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block in initialize**
79:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block in add**
66:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **execute**
40:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **run_callbacks**
78:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/backend/base.rb -> **invoke_job**
230:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block (2 levels) in run**
93:  /usr/lib64/ruby/2.5.0/timeout.rb -> **block in timeout**
103:  /usr/lib64/ruby/2.5.0/timeout.rb -> **timeout**
230:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block in run**
308:  /usr/lib64/ruby/2.5.0/benchmark.rb -> **realtime**
229:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **run**
312:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block in reserve_and_run_one_job**
61:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block in initialize**
66:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **execute**
40:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **run_callbacks**
312:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **reserve_and_run_one_job**
213:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block in work_off**
212:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **times**
212:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **work_off**
175:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block (4 levels) in start**
308:  /usr/lib64/ruby/2.5.0/benchmark.rb -> **realtime**
174:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block (3 levels) in start**
61:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block in initialize**
66:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **execute**
40:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **run_callbacks**
173:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block (2 levels) in start**
172:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **loop**
172:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **block in start**
7:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/plugins/clear_locks.rb -> **block (2 levels) in &lt;class:ClearLocks&gt;**
79:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block (2 levels) in add**
61:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block in initialize**
79:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **block in add**
66:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **execute**
40:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/lifecycle.rb -> **run_callbacks**
171:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/worker.rb -> **start**
137:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/command.rb -> **run**
125:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/command.rb -> **block in run_process**
277:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons/application.rb -> **block in start_proc**
84:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons/daemonize.rb -> **call_as_daemon**
281:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons/application.rb -> **start_proc**
307:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons/application.rb -> **start**
56:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons/controller.rb -> **run**
199:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons.rb -> **block in run_proc**
124:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons/cmdline.rb -> **catch_exceptions**
198:  /GEM_ROOT/gems/daemons-1.4.1/lib/daemons.rb -> **run_proc**
123:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/command.rb -> **run_process**
98:  /GEM_ROOT/gems/delayed_job-4.1.10/lib/delayed/command.rb -> **daemonize**
5:  script/delayed_job.api.rb -> **&lt;top (required)&gt;**
74:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/cli/exec.rb -> **load**
74:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/cli/exec.rb -> **kernel_load**
28:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/cli/exec.rb -> **run**
463:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/cli.rb -> **exec**
27:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/vendor/thor/lib/thor/command.rb -> **run**
126:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/vendor/thor/lib/thor/invocation.rb -> **invoke_command**
387:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/vendor/thor/lib/thor.rb -> **dispatch**
27:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/cli.rb -> **dispatch**
466:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/vendor/thor/lib/thor/base.rb -> **start**
18:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/cli.rb -> **start**
30:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/exe/bundle -> **block in &lt;top (required)&gt;**
124:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/lib/bundler/friendly_errors.rb -> **with_friendly_errors**
22:  /usr/lib64/ruby/gems/2.5.0/gems/bundler-1.17.1/exe/bundle -> **&lt;top (required)&gt;**
23:  /usr/bin/bundle.ruby2.5 -> **load**
23:  /usr/bin/bundle.ruby2.5 -> **&lt;main&gt;**
~~~

## Environment ##
| Key        | Value      |
|------------|------------|
| program_name | delayed_job.1060 |
| HTTP_USER_AGENT |  |

**Comment by hennevogel** [comment]
eeewwww. @coolo this sounds like something the tools should be aware of also right?

**Comment by hennevogel** [comment]
Ah this might be fixed by #12221
